src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.h

```diff
@@ -96,5 +96,7 @@ class HOSTFXR_UTILITY
 
     static
     std::wstring ExpandEnvironmentVariables(const std::wstring & str);
+
+    static const DWORD m_WhereExeProcessTimeoutMS = 10 * 1000;
```

Comment:
what's the motivation to increate the timeout to 10s? should we consider a environment variable for the value if we cannot determine the max timeout value

Reply:
Motivation is that server should not be failing to start on slow machines.

Reply:
Calling where.exe was timing out on our CI (The main reason we saw test failures). We could make the timeout configurable.

Reply:
I don't think we should make it configurable. Maybe we should just increase it to a minute. 

The side effect of where.exe timing out is often app not starting.

Reply:
I meant whether there would be any AV or crash.